rtmbot.py

```diff
@@ -153,26 +161,71 @@ def parse_args():
     )
     return parser.parse_args()
 
+def flag_is_true(flag_str):
+    """ Checks if a string flag should evaluate to True. """
+    flag_str = flag_str.strip().lower()
+    return flag_str == 'true'
```

Comment:
Yeah I actually started writing a thing like that but decided it wasn't worth being flexible about something so simple, so it's either one or the other. Guess I'll just stick with true/True to mimic the YAML file.